src/repl/session.rs

```diff
@@ -679,6 +680,47 @@ impl Session {
         output.append(&mut result);
     }
 
+    #[cfg(feature = "cli")]
+    fn reload(&mut self, output: &mut Vec<String>) {
+        self.asts.clear();
+        self.contracts.clear();
+        self.costs_reports.clear();
+        self.coverage_reports.clear();
+        self.executed.clear();
+        self.initial_contracts_analysis.clear();
+        self.interpreter =
+            ClarityInterpreter::new(
+                self.interpreter.get_tx_sender(),
+                self.settings.repl_settings.clone());
+        let contracts = self.settings.initial_contracts.clone();
+        self.settings.initial_contracts.clear();
```

Comment:
I believe you can avoid this clone by using:
```rust
        let contracts = std::mem::take(&mut self.settings.initial_contracts);
```

Reply:
Very nice. Not only I can replace L965 with `std::mem::take` but I can also get rid of `clear` in L696, according to the [docs](https://doc.rust-lang.org/std/mem/fn.take.html):

```rust
pub fn take<T>(dest: &mut T) -> T 
where
    T: Default, 

// Replaces dest with the default value of T, returning the previous dest value.
```

Reply:
Exactly 👍